src/Bridges/Objective/bridge.jl

```diff
@@ -0,0 +1,118 @@
+"""
+    AbstractBridge
+
+Subtype of [`MathOptInterface.Bridges.AbstractBridge`](@ref) for objective
+bridges.
+"""
+abstract type AbstractBridge <: MOIB.AbstractBridge end
+
+"""
+    bridge_objective(BT::Type{<:AbstractBridge}, model::MOI.ModelLike,
+                     func::MOI.AbstractScalarFunction)
+
+Bridge the objective function `func` using bridge `BT` to `model` and returns
+a bridge object of type `BT`. The bridge type `BT` should be a concrete type,
+that is, all the type parameters of the bridge should be set. Use
+[`concrete_bridge_type`](@ref) to obtain a concrete type for a given function
+type.
+"""
+function bridge_objective(::Type{<:AbstractBridge}, ::MOI.ModelLike,
+                          func::MOI.AbstractScalarFunction)
+    throw(MOI.UnsupportedAttribute(MOI.ObjectiveFunction{typeof(func)}()))
+end
+
+"""
+    function MOI.set(model::MOI.ModelLike, attr::MOI.ObjectiveSense,
+                     bridge::AbstractBridge, sense::MOI.ObjectiveSense)
+
+Return the value of the attribute `attr` of the model `model` for the
+variable bridged by `bridge`.
+"""
+function MOI.set(::MOI.ModelLike, ::MOI.ObjectiveSense,
+                 bridge::AbstractBridge, ::MOI.OptimizationSense)
+    throw(ArgumentError(
+        "Objective bridge of type `$(typeof(bridge))` does not support" *
+        " modifying the objective sense. As a workaround, set the sense to" *
+        " `MOI.FEASIBILITY_SENSE` to clear the objective function and" *
+        " bridges."))
```

Comment:
So what is JuMP going to do? Is it possible to easily set `FEASIBILITY_SENSE` from JuMP?

Reply:
The user can do `set_objective_sense(model, MOI.FEASIBILITY_SENSE)`. This is not ideal but this is a rare use case. We can implement https://github.com/JuliaOpt/MathOptInterface.jl/pull/789#issuecomment-520178883 if this is an issue